src/client/js/controller/welcome.js

```diff
@@ -63,32 +63,29 @@ class Welcome {
+            'manuscript from the <a href="http://leiferlab.princeton.edu/" target="_blank">Leifer Lab</a>: Randi, Sharma, Dvali and Leifer, &#8220Direct measurements',
+            'of functional connectivity in C. elegans reveals contributions from extrasynaptic signaling,&#8221 in',
+            'prep. <b>Only those measured connections with a q-value less than 0.05 are shown.</b> The',
+            'absence of significance does not imply significance of absence.',
             '<p>',
-            'Functional measurements shown in this visualization are from a forthcoming',
-            '(in prep) manuscript from the Leifer Lab:',
+            'Measurements are displayed using <a href="https://nemanode.org/" target="_blank">Nema Node</a>, an open source <a href="https://github.com/zhenlab-ltri/NemaNode" target="_blank">software package</a> described',
+            'in <a href="https://doi.org/10.1038/s41586-021-03778-8" target="_blank">Witvleit et al., 2021</a> from the <a href="https://www.zhenlab.com/" target="_blank">Zhen</a>, <a href="https://scholar.harvard.edu/aravisamuel" target="_blank">Samuel</a>, and <a href="https://lichtmanlab.fas.harvard.edu" target="_blank">Lichtman</a> labs. Select connectomics',
+            'datasets from Witvleit et al, and others are also displayed.',
             '<p>',
-            'Randi, Dvali, Sharma & Leifer, <i>Direct Measurements of Functional Connectivity.</i>',
+            'This project is in beta and has not yet been peer reviewed. Data and code are released under',
+            'a permissive license at <a href="https://github.com/PrincetonUniversity/neuron-graph" target="_blank">https://github.com/PrincetonUniversity/neuron-graph/</a>',
             '<p>',
-            'Only those measured connections with a q-value less than 0.05 are shown.',
-            '<p>',
-            'Measurements are displayed using <b>NemaNode</b> (<i>nemanode.org</i>), an open',
-            'source software package described in <i>Witvleit et al., 2021</i> from the',
-            'Zhen, Samuel and Licthman labs. Select connectomics datasets from',
-            '<i>Witvleit et al</i>, and others, are also displayed.',
-            '<p>',
-            'This project is in beta and has not yet been peer reviewed.',
-            '<p>',
-            'Data and code are released under a permissive license at',
-            '<i>https://github.com/PrincetonUniversity/neuron-graph</i>'
+            'This fork is developed by Research Computing staff in the <a href="https://lsi.princeton.edu" target="_blank">Lewis-Sigler Institute for Integrative',
+            'Genomics</a> (Robert Leach and Lance Parsons) and the <a href="https://pni.princeton.edu" target="_blank">Princeton Neuroscience Institute</a>',
+            '(Benjamin Singer). Request features or report bugs on <a href="https://github.com/PrincetonUniversity/neuron-graph/issues" target="_blank">Github</a>.'
```

Comment:
And since I know we plan to change the repo name, we can note here a spot where it would need to be updated...

Reply:
Good call, added checkbox to #73.